containers/postgresql/docker-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+if [ "$ENV" == "dev" ]; then
+  # prepare-db
```

Comment:
@xmnlab There is dev scripts being copied to `/docker-entrypoint-initdb.d/` but they don't seem to initialize automatically, what you thing would be the best approach to prepare dev db?